go/base/static/vendor/eonasdan-bootstrap-datetimepicker/css/bootstrap-datetimepicker.css

```diff
@@ -0,0 +1,373 @@
+/*!
```

Comment:
No we probably don't. You can see here the line that causes that file to be copied across: https://github.com/praekelt/vumi-go/pull/1359/files#diff-0a08a7565aba4405282251491979bb6bR50 . Let me change that to only take the min version.